contracts/src/libraries/LPMath.sol

```diff
@@ -838,7 +855,7 @@ library LPMath {
                             params.presentValueParams.shareAdjustment,
                             params.presentValueParams.bondReserves,
                             params.presentValueParams.minimumShareReserves,
-                            ONE - _params.presentValueParams.timeStretch,
+                            ONE - params.presentValueParams.timeStretch,
```

Comment:
what happened here?

Reply:
We ran into a `stack-too-deep` issue. We already had a new `params` value, and I just missed this value in previous passes.